code/modules/clothing/suits/bio.dm

```diff
@@ -20,8 +20,8 @@
 	permeability_coefficient = 0.01
 	clothing_flags = THICKMATERIAL
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	slowdown = 1
-	allowed = list(/obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/pen, /obj/item/flashlight/pen, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray)
+	slowdown = 0.5
+	allowed = list(/obj/item/tank/internals, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/glass/beaker)

```

Comment:
i only removed the pen flashlight and that's because the likelihood of it being worth being on your suit storage is far less than the likelihood of it getting in the way

Reply:
if someone wants to "waste" their slot let em

Reply:
ree its back